inc/class-shortcode-ui.php

```diff
@@ -364,4 +367,40 @@ public function handle_ajax_bulk_do_shortcode() {
 		}
 
 	}
+
+	/**
+	 * Decode any attributes
+	 * @param  [type] $out   [description]
+	 * @param  [type] $pairs [description]
+	 * @param  [type] $atts  [description]
+	 * @return [type]        [description]
+	 */
+	function filter_shotcode_atts_decode_encoded( $out, $pairs, $atts ) {
```

Comment:
`filter_shotcode_atts_decode_encoded` should be `filter_shortcode_atts_decode_encoded`

Also, can we mark it explicitly as `public` ?